pkg/scheduler/scheduling_application_test.go

```diff
@@ -89,14 +89,11 @@ func TestAppReservation(t *testing.T) {
 		t.Error("new app should not have reservations for unknown node")
 	}
 
-	queue, err := createRootQueue(nil)
-	if err != nil {
-		t.Fatalf("queue create failed: %v", err)
-	}
+	queue := createDefaultRootQueueProtected(t)
```

Comment:
Same as for the info objects, don't hide the error in a function.
This reversal will also require variable declaration to be reversed to prevent shadowing